common/src/test/java/feast/common/logging/AuditLogEntryTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.common.logging;
+
+import com.google.gson.JsonParser;
+import feast.common.logging.AuditLogEntry.FeastInstance;
+import feast.common.logging.AuditLogEntry.FeastInstance.FeastComponent;
+import feast.common.logging.LogEvent.MessageLogEvent.Direction;
+import feast.common.logging.LogResource.ResourceType;
+import feast.proto.serving.ServingAPIProto.FeatureReference;
+import feast.proto.serving.ServingAPIProto.GetOnlineFeaturesRequest;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+public class AuditLogEntryTest {
+  public List<AuditLogEntry> getTestAuditLogs() {
+    GetOnlineFeaturesRequest requestSpec =
+        GetOnlineFeaturesRequest.newBuilder()
+            .setOmitEntitiesInResponse(false)
+            .addAllFeatures(
+                Arrays.asList(
+                    FeatureReference.newBuilder().setName("feature1").build(),
+                    FeatureReference.newBuilder().setName("feature2").build()))
+            .build();
+
+    return Arrays.asList(
+        AuditLogEntry.newBuilder()
+            .setInstance(FeastInstance.of(FeastComponent.SERVING, "0.6.1"))
+            .setEvent(
+                LogEvent.ofMessage(Direction.INCOMING, "getOnlineFeatures", requestSpec, "adam"))
+            .build(),
+        AuditLogEntry.newBuilder()
+            .setInstance(FeastInstance.of(FeastComponent.CORE, "0.6.1"))
+            .setEvent(
+                LogEvent.ofAction("CREATE", LogResource.of(ResourceType.JOB, "kafka-to-redis")))
+            .build(),
+        AuditLogEntry.newBuilder()
+            .setInstance(FeastInstance.of(FeastComponent.CORE, "0.6.1"))
+            .setEvent(
+                LogEvent.ofTransition(
+                    LogResource.of(ResourceType.FEATURE_SET, "feature_set"), "READY"))
+            .build(),
+        AuditLogEntry.newBuilder()
+            .setInstance(FeastInstance.of(FeastComponent.CORE, "0.6.1"))
+            .setEvent(LogEvent.ofException(new IllegalArgumentException("Bad Argument")))
+            .build());
+  }
+
+  @Test
+  public void shouldReturnStringRepresentationOfAuditLog() {
+    for (AuditLogEntry auditLog : getTestAuditLogs()) {
+      System.out.println(auditLog.toString());
```

Comment:
What is this testing? Wouldnt toString() always return something?

Reply:
Yeah, I guess I was just trying to exercise the code path. Would remove.